honeybee_3dm/face.py

```diff
@@ -0,0 +1,112 @@
+
+# Importing dependencies from Honeybee-3dm package
+from .togeometry import brep_to_face3d, extrusion_to_face3d, mesh_to_face3d
+from .tojson import to_json
+
+# Importing core Honeybee & Ladybug Geometry dependencies
+from honeybee.face import Face
+from honeybee.shade import Shade
+from honeybee.aperture import Aperture
+from honeybee.door import Door
+from honeybee.facetype import face_types
+from honeybee.boundarycondition import boundary_conditions
+from honeybee.typing import clean_and_id_string
+from honeybee.facetype import face_types
+from ladybug_geometry.geometry3d.face import Face3D
+
+
+def to_face(path):
+    """This function looks up a rhino3dm file, converts the objects
+    on the layer name "roof", "wall", "floor", "airwall", "shade", and  "aperture"
+    to Honeybee objects, and writes them to a json file.
+
+    Args:
+        path (A string): The path to the rhino file
+    """
+    # TODO create a quality check for the file path
+
+    # Creating a rhino3dm object from the file at the path provided
+    rhino3dm_file = rhino3dm.File3dm.Read(path)
+    tolerance = rhino3dm_file.Settings.ModelAbsoluteTolerance
+
+    hb_faces = []
+
+    layer_dict = {
+        layer.Name: layer.Index for layer in rhino3dm_file.Layers}
+
+    # A Layer dictionary with layer name : (Honeybee face_type, Class) structure
+    layer_to_hb_object = {
+        'roof': (face_types.roof_ceiling, Face),
+        'wall': (face_types.wall, Face),
+        'floor': (face_types.floor, Face),
+        'airwall': (face_types.air_boundary, Face),
+        'shade': (None, Shade),
+        'aperture': (None, Aperture),
+        'door': (None, Door)}
+
+    for layer in layer_to_hb_object.keys():
+
+        if layer in layer_dict.keys():
+            hb_face_type, hb_face_module = layer_to_hb_object[layer]
+
+            # Gathering planar geometries from the layer
+            rhino_faces = [object for object in rhino3dm_file.Objects
+                           if object.Attributes.LayerIndex == layer_dict[layer]]
+
+            # Creating face names
+            face_names = [geo.Attributes.Name if geo.Attributes.Name else layer +
+                          str(uuid.uuid4())[:8] for geo in rhino_faces]
+
+            # for each rhino geometry gathered, Converting the Rhino3dm geometry
+            # into a Ladybug Face3D objects
+            for count, face in enumerate(rhino_faces):
+                rh_face = face.Geometry
+
+                if rh_face.ObjectType == rhino3dm.ObjectType.Brep:
+                    lb_face = brep_to_face3d(rh_face)
+
+                elif rh_face.ObjectType == rhino3dm.ObjectType.Extrusion:
+                    lb_face = extrusion_to_face3d(rh_face)
+
+                elif rh_face.ObjectType == rhino3dm.ObjectType.Mesh:
+                    lb_face = mesh_to_face3d(rh_face)
+                else:
+                    print(
+                        f'There are objects on layer "{layer}" that this library does not support.')
```

Comment:
You should be more specific here. And I would suggest to use Python's warning module instead of printing.

```suggestion
                    warnings.warn(f'{rh_face.ObjectType} object type is not supported.')
```

Reply:
I have decided to remove this message as we're not concerned if there are rhino3dm objects on a layer that we don't need. We're anyway not selecting those objects.